drools-parser/src/main/java/org/drools/parser/DRLVisitorImpl.java

```diff
@@ -175,7 +176,7 @@ public Object visitLhsPattern(DRLParser.LhsPatternContext ctx) {
             currentPattern.setSource(from);
         }
         Object result = super.visitLhsPattern(ctx);
-            currentConstructStack.peek().addDescr(currentPattern);
+        currentConstructStack.peek().addDescr(currentPattern);
```

Comment:
See my previous comment.
`currentConstructStack` is an instance variable - the logic of this method is strongly based on the assumption that something else happened before so that ` currentConstructStack.peek()` does not returns null: again, I think this is exception-prone

Reply:
@gitgabrio Thanks. I'll address it in DROOLS-7253